test/src/subnet/main.cpp

```diff
@@ -23,107 +23,143 @@
+    string input_data =
+        "../test/data/subnet/subnet_input_test.txt";
+    const char * i_input_data = input_data.c_str();
+
+    std::shared_ptr<cadmium::dynamic::modeling::model> generator =
+        cadmium::dynamic::translate::make_dynamic_atomic_model
+            <ApplicationGen, TIME, const char* >
+                ("generator" , std::move(i_input_data));
+
+
+    /********************************************/
+    /****** SUBNET *******************/
+    /********************************************/
+
+    std::shared_ptr<cadmium::dynamic::modeling::model> subnet1 =
+        cadmium::dynamic::translate::make_dynamic_atomic_model
+            <Subnet, TIME>("subnet1");
+
+
+    /************************/
+    /*******TOP MODEL********/
+    /************************/
+    cadmium::dynamic::modeling::Ports iports_TOP = {};
+    cadmium::dynamic::modeling::Ports oports_TOP = {
+        typeid(outp_out)
+    };
+    cadmium::dynamic::modeling::Models submodels_TOP = {
+        generator, subnet1
+    };
+    cadmium::dynamic::modeling::EICs eics_TOP = {};
+    cadmium::dynamic::modeling::EOCs eocs_TOP = {
+        cadmium::dynamic::translate::make_EOC<subnet_defs::out,outp_out>
+            ("subnet1")
+    };
+    cadmium::dynamic::modeling::ICs ics_TOP = {
+        cadmium::dynamic::translate::make_IC<iestream_input_defs
+            <Message_t>::out,subnet_defs::in>("generator","subnet1")
+    };
+    std::shared_ptr<cadmium::dynamic::modeling::coupled<TIME>> TOP =
+        std::make_shared<cadmium::dynamic::modeling::coupled<TIME>>(
+            "TOP",submodels_TOP,iports_TOP,
+                oports_TOP,eics_TOP,eocs_TOP,ics_TOP);
 
+    ///****************////
 
-/************************/
-/*******TOP MODEL********/
-/************************/
-cadmium::dynamic::modeling::Ports iports_TOP = {};
-cadmium::dynamic::modeling::Ports oports_TOP = {typeid(outp_out)};
-cadmium::dynamic::modeling::Models submodels_TOP = {generator, subnet1};
-cadmium::dynamic::modeling::EICs eics_TOP = {};
-cadmium::dynamic::modeling::EOCs eocs_TOP = {
-  cadmium::dynamic::translate::make_EOC<subnet_defs::out,outp_out>("subnet1")
-};
-cadmium::dynamic::modeling::ICs ics_TOP = {
-  cadmium::dynamic::translate::make_IC<iestream_input_defs<Message_t>::out,subnet_defs::in>("generator","subnet1")
-};
-std::shared_ptr<cadmium::dynamic::modeling::coupled<TIME>> TOP = std::make_shared<cadmium::dynamic::modeling::coupled<TIME>>(
- "TOP", 
- submodels_TOP, 
- iports_TOP, 
- oports_TOP, 
- eics_TOP, 
- eocs_TOP, 
- ics_TOP 
-  );
-
-///****************////
-
-    auto elapsed1 = std::chrono::duration_cast<std::chrono::duration<double, std::ratio<1>>>(hclock::now() - start).count();
-    cout << "Model Created. Elapsed time: " << elapsed1 << "sec" << endl;
+    auto elapsed1 = std::chrono::duration_cast<std::chrono::duration
+        <double, std::ratio<1>>>(hclock::now() - start).count();
+    cout<<"Model Created. Elapsed time: "<<elapsed1<<"sec"<< endl;
```

Comment:
there wont be any space before endl
i.e., cout<<"Model Created. Elapsed time: "<<elapsed1<<"sec"<<endl;

same applies for next cout statement as well

Reply:
solved